lib/Client.js

```diff
@@ -1631,14 +1622,23 @@ class Client extends EventEmitter {
     }
 
     /**
-    * Get a guild's widget object
+    * Get a guild's widget object.
     * @arg {String} guildID The ID of the guild
     * @returns {Promise<Object>} A guild widget object
     */
     getGuildWidget(guildID) {
         return this.requestHandler.request("GET", Endpoints.GUILD_WIDGET(guildID), true);
     }
 
+    /**
+    * Get a guild's widget settings object. Requires MANAGE_GUILD permission
```

Comment:
If we're gonna put required permissions we may as well do it everywhere. Also camelCase